src/renderer/components/settings-general-appearance.tsx

```diff
@@ -193,13 +224,41 @@ export class AppearanceSettings extends React.Component<
             </a>. The easiest way to get started is to clone one of the two existing
             themes and to add your own colors.
           </p>
+          <p>
+            Additionally, if you wish to import an editor theme from VSCode, just pick your JSON file and Fiddle will do the rest.
+          </p>
           <Button
             onClick={this.createNewThemeFromCurrent}
             text='Create theme from current selection'
             icon='duplicate'
           />
+          <Button
+            onClick={this.createNewThemeFromMonaco}
+            text='Add a VSCode Editor theme'
+            icon='duplicate'
+          />
         </Callout>
       </div>
     );
   }
+
+  private async promptForTheme(defaultTheme: LoadedFiddleTheme) {
+    const filePicked = await remote.dialog.showOpenDialog({
+      title: 'Pick a VSCode editor theme file',
+      properties: ['openFile'],
+      filters: [ { name: 'JSON', extensions: ['json']}]
+    });
+    if (filePicked === undefined || filePicked.length === 0) {
+      return Error('Prompt cancelled by user');
```

Comment:
nit: I think returning an `Error` here and returning `false` from the parent function isn't very idiomatic. Could instead simply return early or return `null` and check for `!name` in the caller.